plugins/com.python.pydev.refactoring/src/com/python/pydev/refactoring/tdd/TddCodeGenerationQuickFixParticipant.java

```diff
@@ -267,19 +267,31 @@ private boolean checkMethodCreationAtClass(PyEdit edit, IPyRefactoring pyRefacto
 
     public Definition rebaseAssignDefinition(AssignDefinition assignDef, IPythonNature nature,
             ICompletionCache completionCache) throws Exception {
-        //if the value is currently None, it will be set later on
+        IDefinition[] definitions2;
+        if (assignDef.type.equals("None")) {
```

Comment:
Just to play it safe, please make:

`if ("None".equals(assignDef.type)){`

to avoid a possible null pointer exception (and do the same for the line below: `if (assignDef.value.equals("None")) {`).